Blammo/README.lhs

```diff
@@ -144,7 +144,7 @@ setting the format to `json` will automatically enable it (with
 | ----------- | --------------------------- | ----------------------------------------- |
 | Format      | `setLogSettingsFormat`      | `LOG_FORMAT=tty\|json`                    |
 | Level(s)    | `setLogSettingsLevels`      | `LOG_LEVEL=<level>[,<source:level>,...]`  |
-| Destination | `setLogSettingsDestination` | `LOG_DESTINATION=stdout\|stderr\|@<path>` |
+| Destination | `setLogSettingsDestination` | `LOG_DESTINATION=stdout\|stderr\|null\|@<path>` |
 | Color       | `setLogSettingsColor `      | `LOG_COLOR=auto\|always\|never`           |
 | Breakpoint  | `setLogSettingsBreakpoint`  | `LOG_BREAKPOINT=<number>`                 |
 | Concurrency | `setLogSettingsConcurrency` | `LOG_CONCURRENCY=<number>`                |
```

Comment:
Are you gonna clean up this table

Reply:
Will do. I expected Restyled to, but I guess we don't have prettier-markdown configured in it.

Reply:
> I expected Restyled to, but

I think the `.lhs` plus symlink is why this didn't happen.